[Bug?]: tokens > expose theme primitive color helpers

### Duplicates

- [x] I have searched the existing issues

### Latest version

- [x] I have tested the latest version

### Current behavior 😯

Testing canary yields this error:
```text
. prepare$ panda codegen
│ ✘ [ERROR] No matching export in "node_modules/.pnpm/@jsr+cerberus__tokens@1.7.0-next-f1211ab_…
│     node_modules/.pnpm/@jsr+cerberus__preset-acheron-theme@1.7.0-next-51c15cc_typescript@6.0.…
│       1 │ import { createAcheronPrimitiveColors } from "@jsr/cerberus__tokens";
│         ╵          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
│ ✘ [ERROR] No matching export in "node_modules/.pnpm/@jsr+cerberus__tokens@1.7.0-next-f1211ab_…
│     node_modules/.pnpm/@jsr+cerberus__preset-oceanus-theme@1.7.0-next-51c15cc_typescript@6.0.…
│       1 │ import { createOceanusPrimitiveColors } from "@jsr/cerberus__tokens";
│         ╵          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
│ Build failed with 2 errors:
│ node_modules/.pnpm/@jsr+cerberus__preset-acheron-theme@1.7.0-next-51c15cc_typescript@6.0.3/no…
│ node_modules/.pnpm/@jsr+cerberus__preset-oceanus-theme@1.7.0-next-51c15cc_typescript@6.0.3/no…
```

### Expected behavior 🤔

_No response_

### Steps to reproduce 🕹

Steps:

1.
2.
3.
4.


### Context 🔦

_No response_

### Your environment 🌎

```shell

```

--- a/.changeset/many-guests-kiss.md
+++ b/.changeset/many-guests-kiss.md
@@ -1,0 +1,8 @@
+---
+'@cerberus/preset-acheron-theme': patch
+'@cerberus/preset-elysium-theme': patch
+'@cerberus/preset-oceanus-theme': patch
+'@cerberus/tokens': patch
+---
+
+Update docs

--- a/package.json
+++ b/package.json
@@ -38,5 +38,5 @@
     "bun": ">=1.2.16",
     "node": ">=24.0.0"
   },
-  "packageManager": "pnpm@11.21.0"
+  "packageManager": "pnpm@11.22.0"
 }

--- a/packages/preset-acheron-theme/README.md
+++ b/packages/preset-acheron-theme/README.md
@@ -15,10 +15,7 @@ npx jsr add @cerberus/preset-acheron-theme
 // panda.config.ts
 import { defineConfig } from '@pandacss/dev'
 import { cerberusPreset, cerberusConfig } from '@cerberus/panda-preset'
-import {
-  acheronTheme,
-  getAcheronThemeName,
-} from '@cerberus/preset-acheron-theme'
+import { acheronTheme } from '@cerberus/preset-acheron-theme'
 
 export default defineConfig({
   ...cerberusConfig,
@@ -27,10 +24,6 @@ export default defineConfig({
   exclude: [],
 
   presets: [cerberusPreset(), acheronTheme],
-
-  staticCss: {
-    themes: ['cerberus', getAcheronThemeName()],
-  },
 })
 ```
 

--- a/packages/preset-elysium-theme/README.md
+++ b/packages/preset-elysium-theme/README.md
@@ -15,10 +15,7 @@ npx jsr add @cerberus/preset-elysium-theme
 // panda.config.ts
 import { defineConfig } from '@pandacss/dev'
 import { cerberusPreset, cerberusConfig } from '@cerberus/panda-preset'
-import {
-  elysiumTheme,
-  getElysiumThemeName,
-} from '@cerberus/preset-elysium-theme'
+import { elysiumTheme } from '@cerberus/preset-elysium-theme'
 
 export default defineConfig({
   ...cerberusConfig,
@@ -27,10 +24,6 @@ export default defineConfig({
   exclude: [],
 
   presets: [cerberusPreset(), elysiumTheme],
-
-  staticCss: {
-    themes: ['cerberus', getElysiumThemeName()],
-  },
 })
 ```
 

--- a/packages/preset-oceanus-theme/README.md
+++ b/packages/preset-oceanus-theme/README.md
@@ -15,7 +15,7 @@ npx jsr add @cerberus/preset-oceanus-theme
 // panda.config.ts
 import { defineConfig } from '@pandacss/dev'
 import { cerberusPreset, cerberusConfig } from '@cerberus/panda-preset'
-import { oceanusTheme, getOceanusThemeName } from '@cerberus/preset-oceanus-theme'
+import { oceanusTheme } from '@cerberus/preset-oceanus-theme'
 
 export default defineConfig({
   ...cerberusConfig,
@@ -24,10 +24,6 @@ export default defineConfig({
   exclude: [],
 
   presets: [cerberusPreset(), oceanusTheme],
-
-  staticCss: {
-    themes: ['cerberus', getOceanusThemeName()],
-  },
 })
 ```
 

--- a/packages/tokens/src/factories/primitives.ts
+++ b/packages/tokens/src/factories/primitives.ts
@@ -8,18 +8,34 @@ import { GradientText, GradientValue } from '../theme-contract/theme-contracts.t
  * This module contains factories for creating primitive PandaCSS tokens.
  */
 
+/**
+ * A utility function for building the Cerberus theme primitive colors to register
+ * with PandaCSS.
+ */
 export function createCerberusPrimitiveColors(): NonNullable<Tokens['colors']> {
   return _buildPrimitiveColorsByGroup('cerberus')
 }
 
+/**
+ * A utility function for building the Acheron theme primitive colors to register
+ * with PandaCSS.
+ */
 export function createAcheronPrimitiveColors(): NonNullable<Tokens['colors']> {
   return _buildPrimitiveColorsByGroup('acheron')
 }
 
+/**
+ * A utility function for building the Elysium theme primitive colors to register
+ * with PandaCSS.
+ */
 export function createElysiumPrimitiveColors(): NonNullable<Tokens['colors']> {
   return _buildPrimitiveColorsByGroup('elysium')
 }
 
+/**
+ * A utility function for building the Oceanus theme primitive colors to register
+ * with PandaCSS.
+ */
 export function createOceanusPrimitiveColors(): NonNullable<Tokens['colors']> {
   return _buildPrimitiveColorsByGroup('oceanus')
 }
@@ -54,6 +70,9 @@ export function createPrimitiveSpacing(): NonNullable<Tokens['spacing']> {
   )
 }
 
+/**
+ * A utility function for building radii primitives to register with PandaCSS.
+ */
 export function createPrimitiveRadii(): NonNullable<Tokens['radii']> {
   const mode = primitives.radii.modes.value
 
@@ -78,10 +97,16 @@ export function createPrimitiveRadii(): NonNullable<Tokens['radii']> {
 
 // Getters
 
+/**
+ * Returns the contract gradient token key.
+ */
 export function getContractGradientToken(): GradientText {
   return GRADIENT_TEXT
 }
 
+/**
+ * Returns the list of Cerberus gradients.
+ */
 export function getGradients(): GradientValue[] {
   return GRADIENTS
 }

--- a/packages/tokens/src/types.ts
+++ b/packages/tokens/src/types.ts
@@ -10,8 +10,14 @@ export type NodeMeta = {
 
 // Helpers
 
+/**
+ * Simplifies a type by removing readonly modifiers.
+ */
 export type Simplify<T> = T extends unknown ? { [K in keyof T]: T[K] } : never
 
+/**
+ * Represents a variable color with red, green, blue, and alpha components.
+ */
 export type VariableColor = {
   r: number
   g: number
@@ -19,10 +25,16 @@ export type VariableColor = {
   a: number
 }
 
+/**
+ * Represents a writable type by removing readonly modifiers.
+ */
 export type Writable<T> = {
   -readonly [K in keyof T]: T[K]
 }
 
+/**
+ * Represents a proxy value that can be used to read and write values of type T.
+ */
 export type ProxyValue<T> = {
   <Value>(definition: Value extends T ? Value : T): Value
 } & {

